app/lib/pantry_trak/client.rb

```diff
@@ -69,5 +71,15 @@ def create_reservation_path
         'api/create_freshtrak_reservation_beta.php'
       end
     end
+
+    def handle_unsucessful_response(response)
+      # TODO: In order to rescue the very specific exceptions,
```

Comment:
Is this an actual TODO?

Reply:
It doesn't seem that important that we differentiate the faraday exceptions. Not like we're handling them differently. All we're doing is logging the error.

Reply:
i just thought like if they want to show error messages in future, then we differentiate the faraday exceptions based on the exceptions and then show in UI/logs.
I agree with your point, for now don't need to differentiate the faraday exceptions.